### PR TITLE
Add triage agenda items for waiting on quorum and member requests

### PR DESCRIPTION
**Pull request by peternied**
### Description
During the triage meeting on Aug 27th, based on time constraints member requests were opened up earlier than previously scheduled on the agenda.  This pull request captures this change to ensure meeting members can more quickly discuss an issue/PR.  The opening phase of the triage meeting was not captured, and has been included to document the informal and open tone of the meeting.

### Check List
- [ ] New functionality includes testing
- [X] New functionality has been documented
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: TRIAGING.md
Comment (peternied) [inline TRIAGING.md:1]: Recommended: Use the rich diff view
image

File: TRIAGING.md [hunk 1]
```diff
@@ -20,13 +20,16 @@ If you have an issue you'd like to bring forth please consider getting a link to
 
 ### Is there an agenda for each week?
 
-Meetings are lightly structured as follows:
+Meetings 60 minutes and structured as follows:
+
+1. Wait for quorum: As we gather and waiting for at least three maintainers to join, feel free to turn on video and have informal and open-to-all conversation.  Once quorum is reached, a volunteer will share their screen and proceed with the agenda.
```
Comment (davidlago) [inline TRIAGING.md:25]: What if quorum is never reached? maybe we only have 2 maintainers join. Are we scrapping the meeting? or are there just specific sections that we want to have with enough quorum?

Comment (peternied) [reply TRIAGING.md:25]: I think whoever is in the meeting can make a call what to do or not.  I wouldn't expect the same structure to be honored if this was the case.   Not sure I'd want to capture that in this agenda lines, or do you have a recommendation?

**Review by willyborankin — APPROVED** [review]

File: TRIAGING.md (hunk 1 above)
Comment (davidlago) [reply TRIAGING.md]: I'm not sure we should put anything about quorum at all if we could start w/o reaching one. If this ends up reading "wait for quorum, but if quorum is not reached don't wait", then it's not saying much. If the intent behind this point is more of a "let's give folks a few mins to join", I'd either say that or just remove the paragraph altogether.

File: TRIAGING.md [hunk 2]
```diff
@@ -20,13 +20,16 @@ If you have an issue you'd like to bring forth please consider getting a link to
 
 ### Is there an agenda for each week?
 
-Meetings are lightly structured as follows:
+Meetings is 60 minutes and structured as follows:
```
Comment (davidlago) [inline TRIAGING.md:23]: `s/is/are`

**Review by davidlago — APPROVED** [review]

**Comment by peternied** [comment]
@willyborankin Care to take another look?

**Review by willyborankin — APPROVED** [review]

**Review by reta — APPROVED** [review]

**Pull request merged**